packages/opentelemetry-node/src/NodeTracerProvider.ts

```diff
@@ -51,7 +54,15 @@ export class NodeTracerProvider extends BasicTracerProvider {
       config.contextManager = new AsyncHooksContextManager();
       config.contextManager.enable();
     }
-
     super.register(config);
   }
+
+  getTracer(name: string, version = '*', config?: TracerConfig): NodeTracer {
```

Comment:
Why is a new getTracer being added?

Reply:
As in https://github.com/open-telemetry/opentelemetry-js/pull/1011#discussion_r419112871, implementing `withSpanAsync` on the BasicTracer require to have async hooks so we'll need to have a no-op on the browser.